node-zerox/src/types.ts

```diff
@@ -40,4 +41,13 @@ export interface CompletionArgs {
   maintainFormat: boolean;
   model: ModelOptions;
   priorPage: string;
+  llmParams?: LLMParams;
+}
+
+export interface LLMParams {
+  temperature?: number;
+  top_p?: number;
```

Comment:
Let's keep the casing consistent with the rest of the inputs: `maxTokens`. Also, since we are not using `top_p`, `frequency_penalty` and `presence_penalty`, maybe we shouldn't include them here yet.

Reply:
Our omni repo is using snake case for LLM parameters. Will dm you the url. For other fields maybe users want to tweak? But can definitely delete if you will.

Reply:
That's fine since we are not using the `llmParams` there. Good to keep it consistent within this package.